batchdeobfuscator/batchdeobfuscator.py

```diff
@@ -230,7 +230,7 @@ def execute(self, request: ServiceRequest):
                         file_to_download = re_search(FILE_NAME_REGEX, possible_file_to_download)
                         if file_to_download:
                             # Great, we have a file name that is going to be downloaded.
-                            possible_downloaded_file_name = download_trait["dst"].rsplit("\\\\", 1)[-1]
+                            possible_downloaded_file_name = download_trait["dst"].rsplit("\\", 1)[-1]
```

Comment:
Windows `os.path` functions can be imported as `ntpath` in linux: https://docs.python.org/3.9/library/os.path.html. This looks like a manual implementation of `ntpath.basename`, which should be used instead.